docs/changelog.md

```diff
@@ -5,7 +5,28 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.30.0](https://pypi.org/project/sematic/0.30.0/)
+    * [feature] Grafana dashboards tailored for Sematic installable via Helm
+    * [feature] User metrics SDK
+    * [feature] Add horizontal pod autoscaling and pod disruption budget support
+    * [feature] Health endpoint to display DB health in dashboard
+    * [feature] Add support for set types
+    * [feature] Search for runs in dashboard
```

Comment:
Run search was already a feature. Possibly you saw some of Chance's work on the new dashboard? That's on `main`, but not usable by end-users yet.